lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,37 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server_host: 0.0.0.0
+  dev_server_port: 8080
+  dev_server_https: false
```

Comment:
Unless someone wanna override it in other environment - cloud9? I guess then we would enter into the same problem we discussed earlier right?